### PR TITLE
feat(requestoverview): add link to staging/cms

### PR DESCRIPTION
**Pull request by seaerchin**
corresponding BE PR [here](https://github.com/isomerpages/isomercms-backend/pull/662)

## Problem
This adds the link in the review request to either the staging site or the cms site 

Closes [IS-54]

## Solution
As the link is constructed on teh BE and returned to the FE, this just displays the link in the correct place

## Tests 
- [ ] navigate to a email login based repo
- [ ] create a new image
- [ ] create a new file
- [ ] create a new unlinked page
- [ ] create a new page in folder
- [ ] create a new page in a subfolder
- [ ] create a new resource room page 
- [ ] repeat the above steps but edit existing items rather than creating
- [ ] create a new **empty** folder + subcollection
- [ ] create a new **empty** collection + sub collection
- [ ] delete an item 
- [ ] move an item
- [ ] request a review
- [ ] pages should be able to have a staging + cms link, files/images should not
- [ ] the deleted item should not show up
- [ ] the moved item should show only as a new item

[IS-54]: https://isomeropengov.atlassian.net/browse/IS-54?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kishore03109 — CHANGES_REQUESTED** [review]

File: src/mocks/constants.ts [hunk 1]
```diff
@@ -242,23 +242,26 @@ export const MOCK_ITEMS: EditedItemProps[] = [
     type: ["page"],
     name: "A page",
     path: ["some", "thing"],
-    url: "www.google.com",
+    stagingUrl: "www.google.com",
+    fileUrl: "www.google.com",
```
[the remark this answers is not in the document]
Comment (seaerchin) [reply src/mocks/constants.ts:246]: the naming itself is abit unclear - the `fileUrl` is meant to point to the actual file on the cms but idk if `CmsUrl` would be clearer (open to changing)

this means that the urls are: 
```ts
stagingUrl: staging.amplify.app, 
fileUrl: cms.isomer.gov.sg/
```

[the remark this answers is not in the document]
Comment (kishore03109) [reply src/mocks/constants.ts:246]: just to clarify, this means that the urls are: 

`fileUrl: cms.isomer.gov.sg/path-to-some-file` correct? 
shall we rename this to `fileUrlInCms`?

[the remark this answers is not in the document]
Comment (seaerchin) [reply src/mocks/constants.ts:246]: i think i'll chnage it to `cmsFileUrl` to preserve the `url` suffix for clarityl wdyt

File: src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx [hunk 2]
```diff
@@ -337,26 +336,49 @@ export const RequestOverview = ({
...
-      id: "actions",
+      id: "action",
       header: () => (
         <Th borderBottom="1px solid" borderColor="gray.100" w="10rem" />
       ),
       cell: ({ row }) => (
         <HStack spacing="0.25rem">
           {allowEditing && (
-            <Link href="www.google.com">
+            <Link
```
Comment (harishv7) [inline src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx:346]: @seaerchin lgtm, could you add a screenshot of how this looks like on the UI or add steps to test this locally?

Comment (seaerchin) [reply src/layouts/ReviewRequest/components/RequestOverview/RequestOverview.tsx:346]: there's a story written for this so it should show inside the storybook!

**Review by kishore03109 — COMMENTED** [review]
Chromatic and other CICD builds are failing, may I trouble you to look into them?

File: src/mocks/constants.ts [hunk 3]
```diff
@@ -239,34 +239,30 @@ export const MOCK_SITE_DASHBOARD_INFO: SiteDashboardInfo = {
 
 export const MOCK_ITEMS: EditedItemProps[] = [
   {
-    type: ["page"],
+    type: "page",
```
Comment (harishv7) [inline src/mocks/constants.ts:242]: would be nice if this is a TS type rather than string

**Comment by seaerchin** [comment]
!run e2e

**Comment by seaerchin** [comment]
!run e2e

**Comment by seaerchin** [comment]
!run e2e

**Review by alexanderleegs — APPROVED** [review]
As discussed offline,

- [ ] We should confirm the behaviour of moving files - currently the change only shows up as a new page instead of both a delete and an add
- [ ] Confirm behaviour of delete as well - currently it shows up as a file which cannot be viewed on staging/cms (which makes sense, because it's not available anymore), but is this intuitive to users

Approving first because these are not blocking, but it is behaviour that we should confirm with design!

**Review by kishore03109 — APPROVED** [review]

**Comment by seaerchin** [comment]
> 

filed as IS-229

**Pull request merged**